sky/clouds/cloud.py

```diff
@@ -536,6 +536,10 @@ def get_credential_file_mounts(self) -> Dict[str, str]:
         """
         raise NotImplementedError
 
+    def can_credential_expire(self) -> bool:
```

Comment:
nit:
```suggestion
    def can_credentials_expire(self) -> bool:
```

Reply:
Checks the active credential(only one), the original make sense I think.